server/middleware/checkWhetherToShowServiceOnboardingBanner.ts

```diff
@@ -15,7 +15,7 @@ const checkWhetherToShowServiceOnboardingBanner = async (req: Request, res: Resp
   )
 
   res.locals.showServiceOnboardingBanner =
-    !userRoles.includes(ApplicationRole.ROLE_EDUCATION_WORK_PLAN_EDITOR) &&
+    !userRoles.some(role => Object.keys(ApplicationRole).includes(role)) &&
```

Comment:
Not related to this PR per.se, but in other work I've been getting the old role `ROLE_EDUCATION_WORK_PLAN_EDITOR` removed from people's DPS accounts (replaced with one of the new LWP Contributor or Manager roles as appropriate)
Given the code as it was, now that users dont have the `ROLE_EDUCATION_WORK_PLAN_EDITOR` role, they might incorrectly see the onboarding banner.

I've changed the impl here so that it works in the same way, but rather than working on a single named role, it now works on any of our Application Roles